src/components/expandable-row.vue

```diff
@@ -40,7 +40,7 @@ defineOptions({
   name: 'ExpandableRow'
 });
 
-const expanded = ref(false);
+const expanded = ref(true);
```

Comment:
`ExpandableRow` is also used in `ConnectionToForms`. Should it be expanded by default there as well?

Reply:
No, probably not. There can be a lot of datasets/properties there that would be annoying to have all expanded by default.